plugins/Monolog/Handler/WebNotificationHandler.php

```diff
@@ -58,4 +59,15 @@ protected function write(array $record)
             // Silently ignore the error.
         }
     }
+
+    private function getLiteDebuggingInfo()
+    {
+        $module = Common::getRequestVar('module', false);
+        $action = Common::getRequestVar('action', false);
+        $method = Common::getRequestVar('method', false);
+        $trigger = Common::getRequestVar('trigger', false);
```

Comment:
what is `trigger`? Do you mean archive trigger?

Reply:
Yes, thought it would be useful to see if it's from within an archiving request.

Reply:
Since that could be within a climulti request.

Reply:
👍  makes sense. Wasn't quite sure initially where we use it but then kind of remembered.